### ISSUE TITLE
Use bottom navigation bar

### ISSUE THREAD

**Issue by AndyXheli** [nextcloud/android#11108]
Hi Everyone. I wanted to see what the devs think about changing the Nextcloud Android UI to match the IOS Nextcloud UI. I'm sure this is a lot of work and im sure this will take some time but it sure would be nice to have a cross platform experience for all the users & admins 

![image](https://user-images.githubusercontent.com/59488153/204354975-ed193874-36a6-4ba0-bcd5-2f5a730c1bcd.png)

**Comment by AlvaroBrey** [comment]
Not sure about other people's opinion, but mine is a hard no. Android and iOS have different UI/UX conventions,  this would just make the Android UX worse than it currently is. Not to mention the insane amount of work this would take for negligible benefit.

I'm open to UI changes and improvements (we recently did #11008  for example), but just "make it look like iOS" is a not a good goal.

Anecdotically, I've heard time and time again during my time as an Android dev, people asking for Android apps to look like an iOS app, under the guise of "UX parity" or similar. But strangely, nobody ever asks the iOS devs to make their apps look like Android ones... 

cc @tobiasKaminsky @AndyScherzinger @jancborchardt

**Comment by andyxheli** [comment]
Yes, I agree with you @AlvaroBrey. In my option the only thing that i like about the IOS client is that it has bottom file access menu on the bottom and the only reason why i even brought it up was that i was testing out the Synology drive and Synology photos for someone and they where exactly the same for both ios and android so i thought maybe this would be something that NC wants to do in the future as well. I appreciate the work that was done on https://github.com/nextcloud/android/pull/11008 i really like it.

**Comment by tobiasKaminsky** [comment]
Discussing specific changes, like having a bottom bar (like Google Drive) is something we can really think of.
But having drawer and bottom bar is then also a bit duplicating.

**Comment by AndyScherzinger** [comment]
Drawer and bottom bar is possible, see Google Drive but to be discussed if we want that, of course always "native" from a look&feel, thus needs to be Material 3. Die Drawer menu like iOS, definitely a no for the native look and feel argument and with that, this is already present, given the bottom sheets are also already addressed the only discussion is if we want to add a bottom bar which has been discussed some years ago and decided not to do but can of course be revised. So looking forward to the feedback from  @jancborchardt and @nimishavijay

**Comment by jancborchardt** [comment]
Agree with @AndyScherzinger's assessment. If we look at it high-level, Android and iOS app patterns are all quite similar these days, and of course we have to follow Material Design on Android.

I wouldn't mind a bottom bar in the Android app like in iOS – it allows people to more quickly reach the most common places of the app.

**Comment by andyxheli** [comment]
I want to thank everyone for their feedback. I also want to apologies if I came out like the Android app designed is not good. I love the the designed of the Android app. I should of created the issue stating add menu bottom at the bottom similar to IOS platform or google drive etc. 

For reference  [Navigation bar in Material Design 3](https://m3.material.io/components/navigation-bar)

**Comment by AlvaroBrey** [comment]
I also don't disagree with the bottom bar, but the amount of work it would take it's indeterminate. I'll rename this issue and use it to keep track of the bottom bar only.

**Comment by AndyScherzinger** [comment]
Thanks everyone, so yeah revised decision would now be to implement a navigation bar while this is currently not a priority, so if anybody especially from community is up for it - we are happy to help and side-wing contributors.

**Comment by ZarVladimirII** [comment]
I don´t like the iOS view I see in the opening post, 10-15% of information would not be visible anymore because the space is then preoccupied without adding functionality. I kind of hate that everywhere from webpages to PC-software UI (also overexaggerated spacing). Same Resolution less information at once, especially when there is a previous version to compare to.

For Android the menulist can be opend by wiping from left to right. Yes it got complicated with the introduction of the navigationbar insted of the 3 button-navigation (wiping from the sides is now 'back'). Where it is still possible to open the menu this way is 'DB navigator' and 'Telegram' for example. Yes nextcloud doesn´t support it (Once it did?). This is an easier/faster access method already than hitting the burger-menu-button.
Second, is Nextcloud-App a backup software or a every day software (I don´t see an access to "Talk")? You open and use the UI only once in a while, or there is a synchronisation issue.

So basically, how often do you switch between the view 'Files', 'Favorites', 'Media'? Enough to permanently occupie the viewing space needed for these buttons down there??
For me the iOS screen could use an optimisation, on android I see so much more relevant information.
![Screenshot_20221203-160907](https://user-images.githubusercontent.com/61712819/205447739-eec200ad-daec-44e9-96c0-3d5e27e1c6a5.png)
